src/config-loader/config-loader.ts

```diff
@@ -54,13 +54,13 @@ const getOrCreate = async (
 	uniqueFields: AnyObject,
 	extraFields?: AnyObject,
```

Comment:
Kind reminder that there is also a pinej-client PR about adding a generic `getOrCreate()` method, but I guess we can make it simpler after this.
See: https://github.com/balena-io-modules/pinejs-client-js/pull/100